# move_asound_state_to_var.patch by David B. Harris
#
# Move asound.state location to /var/lib/alsa/
Index: alsa-utils-1.0.18/alsaconf/alsaconf.in
===================================================================
--- alsa-utils-1.0.18.orig/alsaconf/alsaconf.in	2008-10-29 13:42:11.000000000 +0100
+++ alsa-utils-1.0.18/alsaconf/alsaconf.in	2008-11-02 20:59:25.000000000 +0100
@@ -938,8 +938,8 @@
 	  aplay -N $TESTSOUND
       fi
     fi
-    if [ ! -r /etc/asound.state ]; then
-	xecho "Saving the mixer setup used for this in /etc/asound.state."
+    if [ ! -r /var/lib/alsa/asound.state ]; then
+	xecho "Saving the mixer setup used for this in /var/lib/alsa/asound.state."
 	$sbindir/alsactl store
     fi
     clear
Index: alsa-utils-1.0.18/alsactl/alsactl.1
===================================================================
--- alsa-utils-1.0.18.orig/alsactl/alsactl.1	2008-10-29 13:42:11.000000000 +0100
+++ alsa-utils-1.0.18/alsactl/alsactl.1	2008-11-02 20:59:25.000000000 +0100
@@ -43,7 +43,7 @@
 
 .TP
 \fI\-f, \-\-file\fP
-Select the configuration file to use. The default is /etc/asound.state.
+Select the configuration file to use. The default is /var/lib/alsa/asound.state.
 
 .TP
 \fI\-F, \-\-force\fP
@@ -81,7 +81,7 @@
 is used.
 
 .SH FILES
-\fI/etc/asound.state\fP (or whatever file you specify with the
+\fI/var/lib/alsa/asound.state\fP (or whatever file you specify with the
 \fB\-f\fP flag) is used to store current settings for your
 soundcards. The settings include all the usual soundcard mixer
 settings.  More importantly, alsactl is
Index: alsa-utils-1.0.18/alsactl/alsactl.c
===================================================================
--- alsa-utils-1.0.18.orig/alsactl/alsactl.c	2008-10-29 13:42:11.000000000 +0100
+++ alsa-utils-1.0.18/alsactl/alsactl.c	2008-11-02 20:59:25.000000000 +0100
@@ -30,7 +30,7 @@
 #include <alsa/asoundlib.h>
 #include "alsactl.h"
 
-#define SYS_ASOUNDRC "/etc/asound.state"
+#define SYS_ASOUNDRC "/var/lib/alsa/asound.state"
 
 int debugflag = 0;
 int force_restore = 1;
